.github/pull_request_template.md

```diff
@@ -26,4 +27,4 @@ Clearly describe the submitted changes in lots of detail. Include images where h
 
 ### Related Issues
 
-<!-- Does this PR fix a bug or fulfill a Feature Request? Link the related Issues or Feature request here. -->
+<!-- Clearly references the issue it addresses by using `#issue-number` (e.g., `#123`). -->
```

Comment:
has some language erros. Maybe:
"Clearly reference the issue this PR addresses by using '#issue-number' (e.g. ' #123' )